editor/debugger/debug_adapter/debug_adapter_types.h

```diff
@@ -169,10 +169,10 @@ struct Capabilities {
 };
 
 struct Message {
-	int id;
+	int id = 0;
 	String format;
-	bool sendTelemetry = false; // Just in case :)
-	bool showUser;
+	bool sendTelemetry = false;
```

Comment:
There goes my little easter egg :sweat_smile:

Reply:
Brought it back ;)